vendor/github.com/aead/poly1305/poly1305_amd64.s

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2016 Andreas Auernhammer. All rights reserved.
+// Use of this source code is governed by a license that can be
+// found in the LICENSE file.
+
+// +build amd64, !gccgo, !appengine
+
+#include "textflag.h"
+
+DATA ·poly1305Mask<>+0x00(SB)/8, $0x0FFFFFFC0FFFFFFF
+DATA ·poly1305Mask<>+0x08(SB)/8, $0x0FFFFFFC0FFFFFFC
+GLOBL ·poly1305Mask<>(SB), RODATA, $16
+
+// func initialize(state *[7]uint64, key *[32]byte)
+TEXT ·initialize(SB), $0-16
+	MOVQ state+0(FP), DI
+	MOVQ key+8(FP), SI
+
+
+	// state[0...7] is initialized with zero
```

Comment:
Yes, I mean that it is initialized with zeros when `initialize` is invoked (golang memory rule).
Otherwise a `PXOR X0, X0` and `PXOR X1, X1` would be necessary.